src/Services/FileService.php

```diff
@@ -142,14 +142,15 @@ private function loadFile($name, $size, $file, SmartReadFile $smartReadFile, boo
         if ($fileData != null && file_exists($path . $fileData->path . '/' . $fileData->name)) {
             $imageName = $fileData->name;
             $path .= $fileData->path;
-            $mimeType = $fileData->mime;
```

Comment:
This will only return mime object from database. We need the name column instead.